docker-compose.yml

```diff
@@ -67,7 +67,7 @@ services:
       Sitecore_Sitecore__IdentityServer__CertificateRawData: ${SITECORE_ID_CERTIFICATE}
       Sitecore_Sitecore__IdentityServer__PublicOrigin: https://${ID_HOST}
       Sitecore_Sitecore__IdentityServer__CertificateRawDataPassword: ${SITECORE_ID_CERTIFICATE_PASSWORD}
-      Sitecore_License: ${SITECORE_LICENSE}
+      # Sitecore_License: ${SITECORE_LICENSE}
```

Comment:
Could you please also remove `SITECORE_LICENSE` from the main `.env` file and the 2 comment lines above it: https://github.com/Sitecore/Sitecore.Demo.Edge/blob/develop/.env#L66